### ISSUE TITLE
Adding continuous integration tests

### ISSUE THREAD

**Issue by PiJoules** [PiJoules/Septa-Notifier#14]
Since we're posting all these issues, just wanted to throw an idea out there. Do we want to have some sort of automated integration like Travis CI for github or TeamCity at work? I haven't used Travis CI before, but I'd like to make it an end goal to ask a group of people "Who broke the build?"

**Comment by ptrgags** [comment]
Yes! I've tried Travis once before, it's pretty easy to set up. We can discuss this tonight at the club, and we can wait until we have at least one set of unit tests ready to roll to configure it
